<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credit</class>
 <widget class="QWidget" name="credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff> icon.ico</normaloff> icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="4" column="2">
    <widget class="QGroupBox" name="credit_operator_box">
     <property name="minimumSize">
      <size>
       <width>54</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(126, 125, 128, 26);
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-left-radius: 10px;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="calculate">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
 	color: rgb(244, 191, 79);
}</string>
        </property>
        <property name="text">
         <string>calculate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="type">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
 	color: rgb(244, 191, 79);
}</string>
        </property>
        <property name="text">
         <string>annuity</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="munthly_payment_box">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(126, 125, 128, 26);
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-left-radius: 10px;</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="mp">
        <property name="minimumSize">
         <size>
          <width>155</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>155</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);
color: black;</string>
        </property>
        <property name="text">
         <string>monthly payment : </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="show_mp">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: black;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="overpayment_box">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(126, 125, 128, 26);
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-left-radius: 10px;</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="ootl">
        <property name="minimumSize">
         <size>
          <width>155</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>155</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);
color: black;</string>
        </property>
        <property name="text">
         <string>overpayment on the loan : </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="show_pp">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: black;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="total_payout_box">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(126, 125, 128, 26);
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-left-radius: 10px;</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="tp">
        <property name="minimumSize">
         <size>
          <width>155</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>155</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);
color: black;</string>
        </property>
        <property name="midLineWidth">
         <number>-1</number>
        </property>
        <property name="text">
         <string>total payout : </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="show_tp">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: black;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="credit_operand_box">
     <property name="styleSheet">
      <string notr="true">background-color: rgba(126, 125, 128, 26);
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-left-radius: 10px;</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="seven">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="eight">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="nine">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="four">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="five">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="six">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="one">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="two">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="three">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="dot">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="zero">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
    color: rgb(64, 134, 247);
}</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="del">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
}


QPushButton:pressed {
 	color: rgb(236, 105, 95);
}</string>
        </property>
        <property name="text">
         <string>del</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(126, 125, 128, 26);
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
border-top-left-radius: 10px;</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QRadioButton" name="sum">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton{ 
	background-color: None;
}

QRadioButton::indicator::unchecked{ 
	border: 1px solid; 
	border-color: rgba(255, 255, 255, 0);
	border-radius: 7px;
	background-color: white; 
	width: 12px; 
	height: 12px; 
}

QRadioButton::indicator::checked{ 
	border: 1px dotted; 
	border-color: rgba(162, 162, 162);
	border-radius: 7px;
	background-color: rgb(64, 134, 247);
	width: 12px; 
	height: 12px; 
}

</string>
        </property>
        <property name="text">
         <string>sum :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="show_sum">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>15</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: black;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="rate">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton{ 
	background-color: None;
}

QRadioButton::indicator::unchecked{ 
	border: 1px solid; 
	border-color: rgba(255, 255, 255, 0);
	border-radius: 7px;
	background-color: white; 
	width: 12px; 
	height: 12px; 
}

QRadioButton::indicator::checked{ 
	border: 1px dotted; 
	border-color: rgba(162, 162, 162);
	border-radius: 7px;
	background-color: rgb(64, 134, 247);
	width: 12px; 
	height: 12px; 
}

</string>
        </property>
        <property name="text">
         <string>rate</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="show_rate">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: black;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QRadioButton" name="term">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton{ 
	background-color: None;
}

QRadioButton::indicator::unchecked{ 
	border: 1px solid; 
	border-color: rgba(255, 255, 255, 0);
	border-radius: 7px;
	background-color: white; 
	width: 12px; 
	height: 12px; 
}

QRadioButton::indicator::checked{ 
	border: 1px dotted; 
	border-color: rgba(162, 162, 162);
	border-radius: 7px;
	background-color: rgb(64, 134, 247);
	width: 12px; 
	height: 12px; 
}

</string>
        </property>
        <property name="text">
         <string>term</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="show_term">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: black;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
